### PR TITLE
Draft 0.3.4 release

### LINKED ISSUE #16: Add minimum file size (1KB?)

### LINKED ISSUE #21: Claims a plane is not a plane, with 100% certainty that it is a plane.
image

### LINKED ISSUE #22: Results not returning on Safari/iOS
image

### LINKED ISSUE #24: Remove use of Regex to extract UUID from URL and instead pass it as argument to fetchPrediction
See: https://github.com/what-plane/what-plane-frontend/blob/2bd9d90cfa97dbbdaa54660e71e1826a1928bd7e/src/services/predictions.ts#L17

### PR DESCRIPTION
**Pull request by ashrithm**
A draft release of 0.3.4 alpha

Fixed initial bugs:

closes #24 
closes #22 
closes #16 
closes #21

### CODE REVIEW

File: src/components/DisplayResults.tsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ export const DisplayResults = ({
             <Text color="status-warning" size="xsmall">
               <MdError />{" "}
             </Text>
-            Hey, it looks like you've uploaded a picture that doesn't have a
-            plane in it!
+            Hey, it looks like you've uploaded a picture that doesn't have an
+            airplane in it!
```
Comment (Blitheness) [inline src/components/DisplayResults.tsx:35]: Aeroplane for British English?

Comment (ashrithm) [reply src/components/DisplayResults.tsx]: Fixed typo in the [1a3eed0](https://github.com/what-plane/what-plane-frontend/pull/28/commits/1a3eed0a6df6102dc85403b56d6340855623bb5c)

**Review by Will282 — APPROVED** [review]
All good with me!

**Comment by ashrithm** [comment]
v0.3.4 ready :)

**Pull request merged**